index.html

```diff
@@ -1,185 +1,221 @@
-                                                                         this.correctedWords.forEach(word => {
-                                                                                 const existingCorrection = existingCorrections.find(exWord => exword.text === word.text)
-                                                                                 console.log(word);
-                                                                                 newTex = newTex.replace(word.text, `<span class="text-error">${word.text}</span>`)
-                                                        
-                                                                         });
-                                                                         this.richText = newTex
-                                                                         this.event.target.blur();//disable cursor
-                                                                })
-                                                                .catch(error => {
-                                                                        this.answer = 'Error! Could not reach the API. ' + error
-                                                                }).finally( ()=>{
-                                                                          this.loading = false;
-                                                                })
-                                                },
-                                                replaceWord(word) {
-                                                        console.log(this.richText);
-                                                        // replace word 
-                                                        this.richText = this.richText.replaceAll(`<span class="text-error">${word.text}</span>`, word.correct)
-                                                        // remove word from corrected words
-                                                        this.correctedWords = this.correctedWords.filter(correctedWord => correctedWord.correct !== word.correct)
-                                                  
-                                                        console.log(this.richText);
-                                                },
-                                                onInput(event) {
-                                                        const value = event.target.innerText;
-                                                        this.event = event
-                                                        this.text = value;
-                                                },
-                                        },
-                                        watch: {}  
-                                }) 								
-                                
-							 
-				
-							
-                        </script>
-
-        </body>
-
-        <!--End of page -->
-
-</html>
+<!DOCTYPE html/>

```

Comment:
Great one Harry! Well done for fixing those buttons. This is a good addon to our software.

I noticed that you reuploaded the entire HTML file. Well, try and always fix Git to be able to sync with the remote repo so you do not have to reupload files.